### PR TITLE
Task template build

### PR DESCRIPTION
**Pull request by JamesCullen2**
HTML & CSS basic one-page format layout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,8 +2,85 @@
 <html lang="en">
 <head>
     <title>James Cullen Portfolio</title>
+    <link rel="stylesheet" type="text/css" href= "styles.css"/>
 </head>
-<body>
+<body class="style">
```
Comment (kyam-mayden) [inline index.html]: Could you use the 'body' tag as a selector? Also 'style' might cause issue's as may be confused with styles.css

File: index.html [hunk 2]
```diff
@@ -2,8 +2,85 @@
...
+                    </li>
+                    <li class="nav-buttons">
+                        <a href="#about">about</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
 
+        <main class="container">
```
Comment (kyam-mayden) [inline index.html]: Indentation of main is too far in?

File: index.html [hunk 3]
```diff
@@ -2,8 +2,85 @@
...
+                <article>
+                    <h1 id="about">about</h1>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                        incididunt ut labore et dolore magna aliqua. Ut placerat orci nulla
+                        pellentesque. Maecenas pharetra convallis posuere morbi leo urna.
+                        Dignissim sodales ut eu sem integer vitae justo eget magna. Scelerisque
+                        eu ultrices vitae auctor eu.</p>
+                </article>
+            </section>
+
```
Comment (kyam-mayden) [inline index.html]: Not sure of need for gap/blank space

File: index.html [hunk 4]
```diff
@@ -2,8 +2,85 @@
...
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
 
+        <main class="container">
+            <section>
+                <article>
+                    <h1 id="about">about</h1>
```
Comment (kyam-mayden) [inline index.html]: Multiple use of 'about' ID - maybe class is a better selector in this instance? Lines 29, 37 and 45.

File: index.html [hunk 5]
```diff
@@ -2,8 +2,85 @@
...
+            </section>
+        </main>
+    <footer id="contact">
+        <div>
+            <ul>
+                <li class="nav-buttons">
+                    <a href="#">contact</a>
+                </li>
+                <li class="nav-buttons">
+                    <a href="mailto:jamesnac@aol.com">
```
Comment (kyam-mayden) [inline index.html]: Multiple anchor tags within single li item?

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    color: black;
+    text-decoration: none;
+    text-align: center;
+    padding: 10px 0px 10px 0px;
+    background-color: white;
+    border-style: solid;
+    border-color: white;
+}
+
+section {
```
Comment (kyam-mayden) [inline styles.css:51]: Section and header have the same height - could they share a class?

**Review by kyam-mayden — CHANGES_REQUESTED** [review]
A few changes suggested on HTML but CSS is mainly clear & consistent

File: index.html (hunk 1 above)
Comment (JamesCullen2) [reply index.html]: 'Style' class omitted and relevant CSS functions put into body tag a a selector.

File: index.html (hunk 2 above)
Comment (JamesCullen2) [reply index.html]: indent made closer to margin

File: index.html (hunk 3 above)
Comment (JamesCullen2) [reply index.html]: gap omitted

File: index.html (hunk 4 above)
Comment (JamesCullen2) [reply index.html]: using ID in this instance as a link / bookmark to navigate the site on a single page - I have moved the id="about" to the relevant section selector

File: index.html (hunk 5 above)
Comment (JamesCullen2) [reply index.html]: deleted unnecessary anchor tags

File: styles.css (hunk 6 above)
Comment (JamesCullen2) [reply styles.css:51]: I don't think they can at this stage but will try to combine them once I've added more content to see if it fits

**Review by kyam-mayden — APPROVED** [review]
All recommendations implemented

**Pull request merged**